SharpPcap/CaptureEventArgs.cs

```diff
@@ -26,18 +26,29 @@ namespace SharpPcap
     /// <summary>
     /// Capture event arguments
     /// </summary>
-    public class CaptureEventArgs : EventArgs
+    public ref struct CaptureEventArgs
```

Comment:
Just a question: would it be better to make the struct readonly ?

Reply:
Sure, does that improve performance?

Reply:
It prevents compiler from creating defensive copies, whether that improves performance or not, depends on usage

Reply:
That does make sense. That's the impression I got as well, that the compiler could optimize better knowing there were no side effects.